hyperon_das/das.py

```diff
@@ -42,18 +30,16 @@ def __init__(self, **kwargs: Optional[Dict[str, Any]]) -> None:
 
         if query_engine_parameter == 'local':
             self.query_engine = LocalQueryEngine(self.backend, kwargs)
+            logger().info('Initialized local Das')
         elif query_engine_parameter == "remote":
             self.query_engine = RemoteQueryEngine(self.backend, kwargs)
+            logger().info('Initialized remote Das')
```

Comment:
If you put log message only after constructor you will have nothing in the log if the connection itself breaks. I'm OK having this messages here but I saw no log message before the connection is attempted in RemoteQueryEngine. Please make sure that we have proper logging messages telling that a connection is being attempted either here or in the query engine's constructor.

Reply:
@andre-senna look the decorators.py file, in the `retry` function. I put some logs there and also in the `engines.py file on line 206`. notice that I put some logs like `debug`, do you agree with that?

So, the logs in the file can some like this:

[das.log](https://github.com/singnet/das-query-engine/files/13704355/das.log)

Disclaimer: the line with `Starting new HTTP connection`... it's a log of the `request` lib

Reply:
OK